common/benchmark_utils.py

```diff
@@ -133,18 +133,18 @@ def get_all_benchmarks():
     return sorted(all_benchmarks)
 
 
-def get_coverage_benchmarks():
+def get_coverage_benchmarks(benchmarks=get_all_benchmarks()):
```

Comment:
I don't think we should make this change.
There's weird things that happen when you call things in default arguments in python.
https://betterprogramming.pub/the-unexpected-workings-of-pythons-default-arguments-fe1a4addb1dc?gi=32b4d3d6d61f

Reply:
Instead you can do
```
def ...(benchmarks=None):
  if benchmarks is None:
    benchmarks = get_all_benchmarks()
```